### PR TITLE
Updated partners, donors and sponsors

### PR DESCRIPTION
**Pull request by etiennecollin**
In this PR, I:
- Added AI for Good as a partner
- Added Google as a sponsor
- Prepared the info for the MILA as a donor
- Placed the partners, donors and sponsors as follows on the website:
	- Partners: As part of our footer, visible on each page.
	- Donors: listed as planned in the donor section [no change needed].
	- Sponsors: Bottom of main page, no descriptions, just putting all the logos there.

Please test the code as I did everything quite quickly and didn't have time to test everything (such as if the website is still looking good on a phone/smaller screens). From what I tested, everything works.

### CODE REVIEW

**Review by etiennecollin — COMMENTED** [review]
Looks good

File: _includes/partner-logo.html [hunk 1]
```diff
@@ -1,8 +1,7 @@
 <div class="level">
-{% assign partners = site.data.partners | sample: site.data.team.size %}
-{% for partner in partners %} 
-    <a class="level-item has-text-centered" href="{{ partner.link }}">
-      <img src="assets/img/partners/{{ partner.photo }}"  alt="" style="height: 128px;">
-    </a>
-  {% endfor %}
-</div>
\ No newline at end of file
+	{% assign partners = site.data.partners | sample: site.data.team.size %} {% for partner in partners %}
```
Comment (alexhernandezgarcia) [inline _includes/partner-logo.html]: The `sample` part is to randomize the order right? I think for the partners it is better to hard code our preferred order. I will change this.

Comment (etiennecollin) [reply _includes/partner-logo.html]: Good point, thank you!

**Review by alexhernandezgarcia — APPROVED** [review]
Nice work! Ready to merge after a few changes :)

**Pull request merged**